app/playbooks/startup.yml

```diff
@@ -16,19 +16,21 @@
         management_user: "{{ mgmt_user }}"
         management_pass: "{{ mgmt_password }}"
       register: poweron
+      tags:
+        - skip_ansible_lint
```

Comment:
Does ansible lint not like include_roles?

Reply:
it likes it, but it enforces that every task has a name... and includes don't usually have a name and IMO that's ok